lib/run.js

```diff
@@ -85,20 +85,22 @@ var _ = require('lodash'),
         },
         environment: externalLoader.bind(this, 'environment'),
         globals: externalLoader.bind(this, 'globals'),
-        data: externalLoader // todo: use a csv parser to allow csv files.
+        iterationData: externalLoader.bind(this, 'iterationData') // todo: use a csv parser to allow csv files.
```

Comment:
We cannot use `externalLoader` to load iteration data, because it does some unnecessary transformations (for this case). 
Please use `util.fetch` directly.